lib/classes/plugininfo/qbank.php

```diff
@@ -130,4 +130,33 @@ public function load_settings(\part_of_admin_tree $adminroot, $parentnodename, $
             $ADMIN->add($parentnodename, $settings);
         }
     }
+
+    /**
+     * Pre-uninstall hook.
+     *
+     * This is intended for disabling of plugin, some DB table purging, etc.
+     *
+     * NOTE: to be called from uninstall_plugin() only.
+     * @private
+     */
+    public function uninstall_cleanup() {
+        $plugintoremove = $this->displayname;
+        self::remove_unused_column_from_db($plugintoremove);
+    }
+
+    /**
+     * Removes any uninstalled or disabled plugin column in the config_plugins 'qbanksortorder' in core_question.
+     * @param string $plugintoremove Plugin displayed name to remove.
```

Comment:
Can you add a new line before @param thank you.

Reply:
added